plugin.video.presstv/addon.xml

```diff
@@ -15,7 +15,6 @@
     <description lang="en_GB">PRESS TV Addon for Kodi
 Press TV takes revolutionary steps as the first Iranian international news network,broadcasting in English on a round-the-clock basis.</description>
     <platform>all</platform>
-    <language>fa</language>
```

Comment:
why drop the language tag? Shouldn't it be `en` ?

Reply:
Language tag added , thanks for your guidance